index.html

```diff
@@ -30,5 +30,136 @@ <h2>I’am a Software Developer</h2>
+            </ul>
+            <a class="button" href="">See Project</a>
+        </section>
+        <section class="card-works">
+            <h2>Profesional Art Printing Data</h2>
+            <p>
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+            </p>
+            <ul class="tags">
+                <li><a class="tags-elements" href="">html</a></li>
+                <li><a class="tags-elements" href="">bootstrap</a></li>
+                <li><a class="tags-elements" href="">Ruby</a></li>
+            </ul>
+            <a class="button" href="">See Project</a>
+        </section>
+        <section class="card-works">
+            <h2>Profesional Art Printing Data</h2>
+            <p>
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+            </p>
+            <ul class="tags">
+                <li><a class="tags-elements" href="">html</a></li>
+                <li><a class="tags-elements" href="">bootstrap</a></li>
+                <li><a class="tags-elements" href="">Ruby</a></li>
+            </ul>
+            <a class="button" href="">See Project</a>
+        </section>
+        <section class="card-works">
+            <h2>Profesional Art Printing Data</h2>
+            <p>
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+            </p>
+            <ul class="tags">
+                <li><a class="tags-elements" href="">html</a></li>
+                <li><a class="tags-elements" href="">bootstrap</a></li>
+                <li><a class="tags-elements" href="">Ruby</a></li>
+            </ul>
+            <a class="button" href="">See Project</a>
+        </section>
+        <section class="card-works">
+            <h2>Profesional Art Printing Data</h2>
+            <p>
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+            </p>
+            <ul class="tags">
+                <li><a class="tags-elements" href="">html</a></li>
+                <li><a class="tags-elements" href="">bootstrap</a></li>
+                <li><a class="tags-elements" href="">Ruby</a></li>
+            </ul>
+            <a class="button" href="">See Project</a>
+        </section>
+        <section class="card-works">
+            <h2>Profesional Art Printing Data</h2>
+            <p>
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+            </p>
+            <ul class="tags">
+                <li><a class="tags-elements" href="">html</a></li>
+                <li><a class="tags-elements" href="">bootstrap</a></li>
+                <li><a class="tags-elements" href="">Ruby</a></li>
+            </ul>
```

Comment:
- [ ] I'm so impressed with how you used the correct tags `ul` and `li` here 🔥. However, please remove the `a` tag from the lists as the list here is just a list of technologies and not a link 😄.